### ISSUE TITLE
Bug: Kafka messages fail to send due to incompatible send_backoff_ms argument in FastStream with aiokafka 0.11.0

### ISSUE THREAD

**Issue by Siyet** [airtai/faststream#1601]
**Describe the bug**
When using FastStream 0.5.14 with aiokafka 0.11.0 to send messages to Kafka, the operation fails with a `TypeError` due to an unexpected keyword argument `send_backoff_ms` in `AIOKafkaProducer.__init__()`.

**How to reproduce**
Include source code:

```python
from faststream import KafkaBroker
from your_dto_package import PetuhUpdateMessage  # some pydantic v1 model

KAFKA_SERVERS = ["your.kafka.server:port"]  # Update with your Kafka server details

async with KafkaBroker(KAFKA_SERVERS) as broker:
    await broker.publish(
        PetuhUpdateMessage(petuh_id=1),
        topic="petuhs_update",
        key=str(1).encode(),
    )
```

And/Or steps to reproduce the behavior:

1. Ensure aiokafka version 0.11.0 and faststream version 0.5.14 are installed.
2. Execute the provided script to publish messages to Kafka.

**Expected behavior**
Messages should be successfully published to the Kafka topic without any errors.

**Observed behavior**
The operation fails and raises a `TypeError` indicating that `AIOKafkaProducer.__init__()` received an unexpected keyword argument `send_backoff_ms`.

```bash
...
File "/usr/local/lib/python3.10/site-packages/faststream/kafka/broker/broker.py", line 629, in _connect
producer = aiokafka.AIOKafkaProducer(
    TypeError: AIOKafkaProducer.__init__() got an unexpected keyword argument 'send_backoff_ms'
```

**Environment**
- Python version: 3.10.5
- FastStream version: 0.5.14
- aiokafka version: 0.11.0

Downgrading aiokafka to version 0.10.0 resolves the issue, suggesting a compatibility problem with newer versions.

**Additional context**
Downgrading aiokafka to version 0.10.0 as a workaround suggests that changes in aiokafka 0.11.0 are not compatible with the current implementation in FastStream 0.5.14. Further investigation or updates from FastStream may be required to address this incompatibility.

**Comment by Lancetnik** [comment]
Thank you for the report! We already removed this option and the change is in main branch. Wait a bit for today release

**Comment by Lancetnik** [comment]
Closed due 0.5.15 release
